entrypoint

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This command takes one positional argument, `command`, which specifies whether
+to initialize test repos locally, or to start an hg serve comand serving the
+test repos.
+"""
+
+import argparse
+import subprocess
+import os
+from pathlib import Path
+
+
+BASE_REPO_DIR = Path("/repos")
+
+HG_SERV_PORT = os.environ.get("PORT", "8000")
+PHABRICATOR_URI = os.environ.get("PHABRICATOR_URI", "http://phabricator.test")
+
+DO_ARCCONFIG = os.environ.get("ARCCONFIG", "").lower() in ("y", "1", "true")
+BASE_ARC_CONFIG = {
+    "phabricator.uri": PHABRICATOR_URI,
+    "history.immutable": False,
+}
+
+# Add or remove entires from this list to create more or less test repos.
+REPOS = [
+    # At the time of writing, test-repo is used in the "dev" instance for testing.
+    "test-repo",
+    # The repos below are used by the local conduit suite (phabricator and lando-api).
+    # If these values are changed here, the local phabricator SQL dump and the hardcoded
+    # repo values in `landoapi.repos` will need to be updated.
+    "first-repo",
+    "second-repo",
+    "third-repo",
+]
+
+# Create a config template for the `hg serve` command.
+WEBDIR_PATHS = "\n".join([f"{r} = {BASE_REPO_DIR/r}" for r in REPOS])
+WEBDIR_CONF = "[paths]\n{}".format(WEBDIR_PATHS)
```

Comment:
You can just do the following instead:
```ini
[paths]
/ = ${BASE_REPO_DIR}/*
```
Might have the syntax for string variable formatting slightly wrong but you get the point. :)

https://hg.mozilla.org/hgcustom/version-control-tools/file/763969c51d17c5324a894aa37b4ce48f43c66b68/testing/firefoxrepos.sh#l36